Story 6271 lease handling

primiti opened the pull request:
This is some cleanup being addressed before doing the real work on this story.    This makes the following changes:

* Updates the references in Gemfile.lock for the gems with security vulnerabilities flagged by github.  These were only used during test runs, but it should get rid of the warning messages.
* Updates rubocop file to use company standard.
* Addresses all rubocop comments.
* Adds rubocop to standard test run.

omeedrabani commented on test/dummy/test/unit/passport_test.rb:
```diff
@@ -66,12 +68,12 @@ class PassportTest < ActiveSupport::TestCase
-          "city"=>"Santa Barbara",
-          "foreign_visits"=>[{"country"=>"Canada"},{"country"=>"Mexico"}],
-          "gender"=>"female",
+          "birthdate" => Time.parse("2011-8-11"),
```
nit: align `=`'s

omeedrabani commented on test/unit/aggregate_store_test.rb:
```diff
@@ -18,13 +20,13 @@ def add(attr, message)
   context "aggregate_attribute" do
     setup do
-      @store = Class.new {}
+      @store = Class.new { }
```
Shouldn't it be `{}`?

omeedrabani replied:
Same question for the other instances of this as well

primiti replied:
That was a rubocop autocorrect change....  I am inclined to not fight it.

primiti replied:
Actually, no, this is an empty block being passed, not an empty hash, so the spaces are right.

omeedrabani commented on test/unit/attribute/date_time_test.rb:
```diff
@@ -17,7 +19,8 @@ class Aggregate::Attribute::DateTimeTest < ActiveSupport::TestCase
   should "Load into users timezone, store the same regardless of timezone" do
     user_time = Time.zone.local(2008, 3, 10).in_time_zone("Eastern Time (US & Canada)")
     begin
-      old_time_zone, Time.zone = Time.zone, "Eastern Time (US & Canada)"
```
Wow, this was hard to read.

omeedrabani commented on test/unit/container_test.rb:
```diff
@@ -555,16 +560,16 @@ def fixup1
+              "tracking_number" => "1245",
+              "weight_in_ounces" => 5,
+              "shipping_method" => "UsPostal",
+              "signature_required" => true
```
nit: align `=>`'s

omeedrabani commented on test/unit/container_test.rb:
```diff
@@ -660,16 +667,16 @@ def fixup1
+              "tracking_number" => "1245",
+              "weight_in_ounces" => 5,
+              "shipping_method" => "UsPostal",
+              "postage_due" => '50.2'
```
nit: align `=>`'s

primiti commented:
Thanks for the review.  I pushed formatting fixes.

primiti commented:
Oh, damn, I didn't.  I pushed the changes to a branch off of this one that I am having Alec review.  Can you trust me that I addressed your comments and give this a 👍 ?
